.github/workflows/unit_tests.yml

```diff
@@ -27,6 +28,13 @@ jobs:
           pip install -e .[test]
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
       - name: Test with pytest
+        env: # Configure endpoints
+          AZUREAI_CHAT_ENDPOINT: ${{ secrets.AZUREAI_CHAT_ENDPOINT }}
```

Comment:
These work in my fork, but of course would have to be updated in the `guidance-ai/guidance` repo.